crates/ty_ide/src/goto_type_definition.rs

```diff
@@ -226,24 +226,7 @@ mod tests {
             "#,
         );
 
-        assert_snapshot!(test.goto_type_definition(), @r#"
-        info[goto-type-definition]: Type definition
-           --> stdlib/builtins.pyi:892:7
-            |
-        890 |     def __getitem__(self, key: int, /) -> str | int | None: ...
-        891 |
-        892 | class str(Sequence[str]):
-            |       ^^^
-        893 |     """str(object='') -> str
-        894 |     str(bytes_or_buffer[, encoding[, errors]]) -> str
-            |
-        info: Source
-         --> main.py:2:22
-          |
-        2 |             a: str = "test"
-          |                      ^^^^^^
-          |
-        "#);
+        assert_snapshot!(test.goto_type_definition(), @"No goto target found");
```

Comment:
This is the main Problematic thing about this implementation -- we don't ask the type checker if this string actually was a string annotation.